apisix/admin/consumers.lua

```diff
@@ -73,14 +70,14 @@ local function check_conf(username, conf, need_username, schema)
     end
 
     if conf.plugins then
-        ok, err = plugins.check_schema(conf.plugins, core.schema.TYPE_CONSUMER)
+      local ok, err = check_duplicate_key(conf.username, conf.plugins)
```

Comment:
why change the order of those checks? I think we should keep checking static schema first indeed

Reply:
The plugin's configuration will be encrypted during check_schema, and will need to be decrypted later.